backend/layers/business/business.py

```diff
@@ -684,6 +684,12 @@ def ingest_dataset(
         self.database_provider.update_dataset_processing_status(
             new_dataset_version.version_id, DatasetProcessingStatus.INITIALIZED
         )
+        self.update_dataset_version_status(
+            new_dataset_version.version_id,
+            DatasetStatusKey.ATAC_FRAGMENT,
+            DatasetConversionStatus.SKIPPED,
+            # TODO: remove when atac is supported
+        )
```

Comment:
my instinct is that this should be "NA" rather than SKIPPED by default for non-atac-seq assays

Reply:
Keeping as skipped for now so functional tests will pass since they expect at least "Skipped"

Reply:
isn't that just a one-line update [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7466/files/6c9067d38f582ef4ca26b0b013138e3788f479ef#diff-ba8cca093fba034995f714de1f23604b1ab52d2bcdc7b9181428ec5339f615aaR119)?